server/Cargo.toml

```diff
@@ -10,7 +10,13 @@ datafusion = "10.0.0"
 datafusion-physical-expr = "10.0.0"
 object_store = "0.3.0"
 
+
```

Comment:
Fixed

Reply:
@skejserjensen Should we group dependencies and if so, how should they be grouped?

Reply:
I simply grouped them based on their functionality because it seemed intuitive to me, e.g., `arrow-flight` and `tonic` together because `arrow-flight` uses `tonic` for gRPC. However, if @agneborn98 and you (@CGodiksen) think another ordering i better, e.g., a simple sorted list, then that is fine by me.

Reply:
That grouping seems fine, lets keep doing that.